azure-pipelines.yml

```diff
@@ -95,24 +98,56 @@ stages:
           secureFile: SonarSourceSecret.snk
 
       - task: DownloadSecureFile@1
-        displayName: 'Download pfx file'
-        name: pfx
+        displayName: 'Download p12 file'
+        name: SM_CLIENT_CERT
         inputs:
-          secureFile: 'SonarSource-2021-2023.pfx'
+          secureFile: digicert_authentication_certificate.p12
+ 
+      - task: DownloadSecureFile@1
+        displayName: 'Download crt file'
+        name: SM_CLIENT_CRT
+        inputs:
+          secureFile: cert_525594307.crt
+
+      - task: PowerShell@2
+        displayName: "Signing certificate setup"
+        # ToDo: add back the condition
+        # condition: eq(variables.isReleaseBranch, 'True')
+        env:
+          SIGNTOOL_PATH: $(SignToolPath)
+          SM_CLIENT_CERT_FILE: $(SM_CLIENT_CERT.secureFilePath)
+          SM_CLIENT_CERT_PASSWORD: $(SM_CLIENT_CERT_PASSWORD)
+          SM_API_KEY: $(SM_API_KEY)
+          SM_CERT: $(SM_CERT)
+          SM_CERT_FP: $(SM_CERT_FP)
+          SM_HOST: $(SM_HOST)
+          SM_KP: $(SM_KP)
```

Comment:
You can remove the values that are not secret in the variable group as they should be available automatically.